test/integration/scripts/test_pklfix.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2015-2025 Earth Sciences Department, BSC-CNS
+#
+# This file is part of Autosubmit.
+#
+# Autosubmit is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Autosubmit is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Autosubmit.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Test for the autosubmit pklfix command"""
+
+from pathlib import Path
+from unittest.mock import patch
```

Comment:
Can you replace this by the pytest fixture. Most of the code is already using that one, so it makes it more uniform and avoids any issues in case `pytest` changes anything related to mockers/mocking.